src/components/pages/lessons/overlay/add-note-overlay.tsx

```diff
@@ -1,15 +1,32 @@
 import * as React from 'react'
 import {Switch} from '@headlessui/react'
 import VisuallyHidden from '@reach/visually-hidden'
+import axios from 'axios'
 
-const AddNoteOverlay: React.FC<{onClose: any}> = ({onClose}) => {
+const AddNoteOverlay: React.FC<{
+  onClose: any
+  resourceId: string
+  currentTime: number
+}> = ({onClose, resourceId, currentTime}) => {
   const [enabled, setEnabled] = React.useState(false)
   const inputRef = React.useRef<HTMLTextAreaElement>(null)
   React.useEffect(() => {
     if (inputRef) {
       inputRef.current?.focus({preventScroll: true})
     }
   }, [])
+
+  const addNote = () => {
+    if (inputRef.current) {
+      axios
+        .post(`/api/lessons/notes/${resourceId}`, {
+          text: inputRef.current.value,
+          startTime: currentTime,
+        })
+        .then(({data}) => onClose(data.data))
```

Comment:
send back our new note data when we close the modal. it's actually an array with a single item